tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -111,12 +179,70 @@ Object.Pipeline {
 				out_ch_map	$CHANNEL_MAP_3_POINT_1
 			}
 		}
+
+		Object.Widget.copier."2" {
+			stream_name $DMIC0_NAME
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
```

Comment:
@tmleman @marcinszkudlinski  @softwarecki @ArsenEloglian do we need to change dma buffer size to 4 * ibs  due to https://github.com/thesofproject/sof/pull/6835 ?